starport_template/pubspec.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
     git:
       url: https://github.com/tendermint/flutter.git
       path: packages/cosmos_ui_components
-      ref: assets_portfolio
+      ref: account-listing-bottom-sheet
```

Comment:
Will fix this back to `starport-redesign` after this PR is merged.

Reply:
you can always split your work into two prs, one that updates only `cosmos_ui_components` and one that updates the `starport_template`. something like #128 and #129

Reply:
My bad. Will do it from the next PRs.